### PR TITLE
Add com.jetpackduba.Gitnuro

### PR DESCRIPTION
**Pull request by JetpackDuba**
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. 
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://github.com/JetpackDuba/Gitnuro/
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.jetpackduba.Gitnuro

File: com.jetpackduba.Gitnuro.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - type: file
+        url: https://github.com/JetpackDuba/Gitnuro/releases/download/v0.1.0/Gitnuro-0.1.0.jar
+        sha256: 9502032cf103b9a31bdaf1a8f0495d23917ecac38b43a2c34bb882f070b47a63
+      - type: file
+        path: com.jetpackduba.Gitnuro.desktop
+      - type: file
+        path: icon.png
+      - type: file
+        path: com.jetpackduba.Gitnuro.metainfo.xml
+finish-args:
```
Comment (hfiguiere) [inline com.jetpackduba.Gitnuro.yml:33]: this is usually above "modules" as a common practice

File: com.jetpackduba.Gitnuro.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>com.jetpackduba.Gitnuro</id>
```
Comment (hfiguiere) [inline com.jetpackduba.Gitnuro.metainfo.xml:3]: `http://jetpackduba.com/` redirect to an unconfigured website. is this app-id really valid?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.jetpackduba.Gitnuro.metainfo.xml (hunk 2 above)
Comment (JetpackDuba) [reply com.jetpackduba.Gitnuro.metainfo.xml:3]: Yeah, I've just bought the domain this week, I'm currently working on the website.

File: com.jetpackduba.Gitnuro.yml (hunk 1 above)
Comment (JetpackDuba) [reply com.jetpackduba.Gitnuro.yml:33]: Do you mean the `finish-args`?

File: com.jetpackduba.Gitnuro.metainfo.xml (hunk 2 above)
Comment (JetpackDuba) [reply com.jetpackduba.Gitnuro.metainfo.xml:3]: gitnuro.jetpackduba.com is already up! (Still waiting for Gitlab pages to setup the SSL cert tho)

**Comment by JetpackDuba** [comment]
bot, build com.jetpackduba.Gitnuro

**Pull request closed without merging**